src/main/java/com/microsoft/appcenter/appium/Event.java

```diff
@@ -12,6 +26,9 @@
     }
 
     public static Event createLabel(String id, String testName, String className, int run, String label, String screenshotPath, int screenshotOrientation, boolean screenshotRotated) {
+        if (null == label || label.length() < 1 || label.length() > MAX_LABEL_SIZE) {
+            throw new IllegalArgumentException("Labels must be non-empty strings of length <= " + MAX_LABEL_SIZE);
+        }
```

Comment:
Didn't we decide to truncate when `label.length() > MAX_LABEL_SIZE` and not raise an exception? Although I can see that max length is now 256 which is more than enough for anyone!!! ;-P